dvc/fs/fsspec_wrapper.py

```diff
@@ -19,8 +19,14 @@ def _with_bucket(self, path):
         return path
 
     def _strip_bucket(self, entry):
-        _, entry = entry.split("/", 1)
-        return entry
+        try:
+            bucket, path = entry.split("/", 1)
+        except ValueError:
+            # If there is no path attached, only returns
+            # the bucket (top-level).
+            bucket, path = entry, None
+
+        return path or bucket
```

Comment:
this is just a precaution regarding a case where path might be empty (`bucket/`) but the split still succeeds. Another replacement might be `if '/' not in entry.rstrip('/'): ...` though the code above made more sense to me